metis-enrichment/metis-enrichment-client/src/main/java/eu/europeana/enrichment/rest/client/EnrichmentWorker.java

```diff
@@ -0,0 +1,362 @@
+package eu.europeana.enrichment.rest.client;
+
+import java.io.UnsupportedEncodingException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Set;
+import org.jibx.runtime.JiBXException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import eu.europeana.corelib.definitions.jibx.IsPartOf;
+import eu.europeana.corelib.definitions.jibx.PlaceType;
+import eu.europeana.corelib.definitions.jibx.RDF;
+import eu.europeana.corelib.definitions.jibx.ResourceOrLiteralType;
+import eu.europeana.enrichment.api.exceptions.UnknownException;
+import eu.europeana.enrichment.api.external.model.EnrichmentBase;
+import eu.europeana.enrichment.api.external.model.EnrichmentResultList;
+import eu.europeana.enrichment.utils.EnrichmentUtils;
+import eu.europeana.enrichment.utils.InputValue;
+import eu.europeana.metis.dereference.DereferenceUtils;
+import eu.europeana.metis.dereference.client.DereferenceClient;
+
+/*
+ * To be integrated into eCloud topology. This code serves as an example of enrichment through use of the
+ * enrichment and dereferencing clients and utils. 
+ * 
+ */
+public class EnrichmentWorker {
```

Comment:
This class has SonarQube issues (beyond the use of System.out.println) that should be solved.

Reply:
Not worth the time investment. This class is just a placeholder/container for the logic that will be implemented in the eCloud topology.